pkg/provisioningv2/rke2/planner/encryptionkeyrotation.go

```diff
@@ -29,8 +28,6 @@ const (
 	encryptionKeyRotationSecretsEncryptApplyCommand  = "secrets-encrypt-apply"
 	encryptionKeyRotationSecretsEncryptStatusCommand = "secrets-encrypt-status"
 
-	encryptionKeyRotationLeaderAnnotation = "rke.cattle.io/encrypt-key-rotation-leader"
```

Comment:
what happens if a user updates from an annotation version of Rancher to a status version of Rancher in the middle of an encryption key rotation?